### ISSUE TITLE
All date input fields should be controlled by calendar tool

### ISSUE THREAD

**Issue by cisaacs01** [instedd/cdx#840]
Do not allow users the possibility to mistake international date formats by using a calendar selection tool

**Comment by nditada** [comment]
@anthonyoleary or @tripledub are you working on this already?

**Comment by tripledub** [comment]
Yes we are, although @nditada did mention there is possibly a component for the date picker that Jonathan has devised previously. Either way, the three of us can work it out between us

**Comment by cisaacs01** [comment]
Being handled by UK team

**Comment by pmallol** [comment]
As for Version: 0.13-pre1, only the Date input in the Home page is controlled by a calendar tool. Other Date filters still maintain the 'Previous Month/Week/year' options.
It is expected all Date filters to be controlled by calendar tools.

**Comment by straight-shoota** [comment]
This is still an issue with date fields on the sample form, for example. The fields on the sample filter form have widgets, though.

**Comment by straight-shoota** [comment]
Also, fields like `date_produced` could probably be pre-filled with the current date for easy use.
